p2p/peering.go

```diff
@@ -119,6 +119,37 @@ type LocalPeer struct {
 	rpMu   sync.Mutex
 }
 
+type SnapShot struct {
+	Id         uint64
+	Services   wire.ServiceFlag
+	InitHeight int32
+	Raddr      string
+	Na         *wire.NetAddress
```

Comment:
The remote address and net address are the same address.  No reason to have both.  Since the services are already included in the struct, just the remote address can be included.

Reply:
fixed